### ISSUE TITLE
Unable To Import CSV

### ISSUE THREAD

**Issue by thecodingrasta** [knadh/listmonk#1201]
**Version:**
 - listmonk: v2.3.0
 - OS: Docker (Railway App)

**Description of the bug and steps to reproduce:**
I am attempting to import a CSV file which is encoded in UTF-8. I select the file but the upload button remains greyed out. I am following the Raw CSV Example supplied.

Example:
email,name,attributes
user1@mail.com,"User One","{""age"": 42, ""planet"": ""Mars""}"
user2@mail.com,"User Two","{""age"": 24, ""job"": ""Time Traveller""}" 

My CSV:
email,name,attributes
test@test.com,"Test Data","{""company"": ""Test Company"", ""addressLine1"": ""Address Line 1"", ""addressLine2"": ""Address Line 2"", ""addressCity"": ""Address City"", ""addressState"": ""Address State"", ""addressPostal"": ""POSTAL"", ""addressRegion"": ""Region"", ""landline"": ""01234 567890"", ""jobTitle"": ""Job Title"", ""mobile"": ""07213 666342"", ""linkedInURL"": ""https://www.linkedin.com""}"

As no errors are reported when I attempt to import the file I have no idea what may be wrong. Ideally the upload feature should report this.

**Comment by knadh** [comment]
Does the import say that it completely successfully if there are no errors? What exactly do you see?

**Comment by thecodingrasta** [comment]
@knadh Thanks for your response. I select the file but the data doesn't seem to be imported. It just shows the name of the file with the option to remove it. Please see screenshot, as you can see the Upload button is greyed out and cannot be clicked.

![image](https://user-images.githubusercontent.com/102020625/220637654-7491fdef-b10f-4624-b57f-3eb659250282.png)

**Comment by knadh** [comment]
Looks like you haven't selected the lists in the `Lists` field above. The `Upload` button becomes actively only once you select the lists to import to.

**Comment by thecodingrasta** [comment]
@knadh I have now selected a list but I am still unable to import my CSV file. It just sits at 0/70 and doesn't do anything. No errors reported at all. Can you re-open this issue until I am able to import my CSV file please.

![image](https://user-images.githubusercontent.com/102020625/220900645-235465ef-a97b-4e17-bec9-1bae4695572c.png)

**Comment by thecodingrasta** [comment]
@knadh I also found another issue. When selecting 'Stop Import' and navigating away from the Import page, when I attempt to import a CSV file again I receive the below:

![image](https://user-images.githubusercontent.com/102020625/220901349-77fc8ef1-63ec-483a-8305-ae89a2df755b.png)

Your CSV Import feature appears to be very buggy. I am following the examples provided and the encoding is UTF-8 but am still unable to import my contacts.
